tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_browser_Browser.java

```diff
@@ -1045,6 +1033,17 @@ public void test_TitleListener_event() {
 	assertTrue(errMsg, passed);
 }
 
+@Test
+public void test_isLocationForCustomText() {
+	String expectedTitle = "Website Title";
+	Runnable browserSetFunc = () -> {
+		String html = "<html><title>Website Title</title><body>Html page with custom title</body></html>";
+		boolean opSuccess = browser.setText(html);
+		assertTrue("Expecting setText() to return true", opSuccess);
+	};
+	validateTitleChanged(expectedTitle, browserSetFunc);
+	assertTrue("Browser didn't navigate to the implicit custom text location", browser.isLocationForCustomText(browser.getUrl()));
+}
```

Comment:
Can't the test method be already implemented like this in the original contribution in #1409 rather than changing it in this PR directly after introducing it? Or is there a reason why it must be implemented differently before the changes of this PR?

Reply:
Done

Reply:
Not this change seems to be gone completely. Isn't is relevant anymore?

Reply:
I have moved it all to #1409 since it just adds an extra check of cutom tect url on setText which is relevant to the previous PR.

Reply:
I see. Then please see my comment on #1409:
https://github.com/eclipse-platform/eclipse.platform.swt/pull/1409#discussion_r1740776474

Reply:
Addressed